STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/service/UserManagementService.java

```diff
@@ -181,14 +184,14 @@ public boolean isUserOnline(String username) {
 
     private org.openl.rules.security.User createSecurityUser(User user) {
         return SimpleUser.builder()
-                .setFirstName(user.getFirstName())
-                .setLastName(user.getSurname())
-                .setUsername(user.getLoginName())
-                .setPasswordHash(user.getPasswordHash())
-                .setPrivileges(PrivilegesEvaluator.createPrivileges(user))
-                .setEmail(user.getEmail())
-                .setDisplayName(user.getDisplayName())
-                .setExternalFlags(user.getUserExternalFlags())
-                .build();
+            .setFirstName(user.getFirstName())
+            .setLastName(user.getSurname())
+            .setUsername(user.getLoginName())
+            .setPasswordHash(user.getPasswordHash())
+            .setPrivileges(PrivilegesEvaluator.createPrivileges(user))
+            .setEmail(user.getEmail())
+            .setDisplayName(user.getDisplayName())
+            .setExternalFlags(user.getUserExternalFlags())
+            .build();

```

Comment:
Return formatting as it was. Implementation should not have formatting due complexity of unnecessary branches merging. Formatting of the code should be performed as the separate task to minimize conflicts.